homeassistant/components/device_automation/__init__.py

```diff
@@ -121,6 +140,47 @@
     return automations
 
 
+def _custom_serializer(schema):
```

Comment:
Let's not keep this local to here but just add it to `config_validation`. You don't need to support everything at once, just this one thing you want to support.

Reply:
Fixed.